src/components/NumberInput/NumberInput.js

```diff
@@ -4,6 +4,7 @@ import { iconCaretUp, iconCaretDown } from 'carbon-icons';
 import Icon from '../Icon';
 import classNames from 'classnames';
 import { settings } from 'carbon-components';
+import { componentsX } from '../../internal/FeatureFlags';
```

Comment:
Wondering if you can port `breakingChangesX` flag over this PR and use it, given the change doesn't seem to be tied to experimental theme: https://github.com/IBM/carbon-components-react/pull/1606/files#diff-860a64fa9638c1ac3f8ec09230fc26a9R21